insanely_fast_whisper_rocm/core/integrations/__init__.py

```
@@ -1,3 +1,35 @@
+
+Each integration is imported lazily by its consumers to avoid pulling
+optional dependencies (like pyannote.audio) at module load time.
+"""
 
 from .stable_ts import stabilize_timestamps  # noqa: F401
+
+__all__ = [  # noqa: F822  # diarize/clear_diarization_cache via __getattr__
+    "stabilize_timestamps",
+    "diarize",
+    "clear_diarization_cache",
+]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)